public/assets/css/homepage.css

```diff
@@ -71,10 +69,4 @@ body{
 	margin-top: 20px;
 	display: inline-block;
 	color: #fff;
-
```

Comment:
Great Job!  on Line 23, the body tag's linear gradient, would you mind changing the colors? Orange and white pops out a little too much, and with white text, any text lower than the contact us button will be invisible